src/grammar.coffee

```diff
@@ -836,7 +836,7 @@ grammar =
   ]
 
   Operation: [
-    o 'UNARY Expression',                       -> new Op $1 , $2
+    o 'UNARY Expression',                       -> new Op $1.toString(), $2, null, null, originalOperator: $1.original
```

Comment:
Rather than passing `null` to skip arguments, we should pass `undefined`, as a general rule. Since only `undefined` allows function parameter default values to be set.

Reply:
updated to `undefined` (in all the updated rules)